Mod1/Lessons/Week5/Scope.md

```diff
@@ -0,0 +1,232 @@
+# Scope and Access Modifiers
+
+## Learning Goals
+* Identify how **scope** affects where variables can be accessed
+* Have a reference to look at when **scope** errors show up
+* Understand the difference between the **public** and **private** access modifiers
+
+# Scope
+
+## Intro
+Scope is what you have access to and where you have access to it.
+
+Scope is the kind of thing that you think you don’t need to know until it starts causing you trouble. Even then, it is generally unnecessary to know the full rules. It’s more important that when you see unexpected behavior scope of your variables/method names is one of the things on your list to check.
+
+## Examples of Scope Errors
+
+You've probably already run into scope errors.
+
+Maybe you've seen something like this where you already defined a variable but are still getting an error saying your variable doesn't exist.
+
+Example 1:
+![Scope Example 1](/Mod1/Images/Week5/ScopeExample1.png)
+
+Or maybe you've tried to use the variable i multiple times in your program and found that you need to create a new variable each time.
+
+Example 2:
+![Scope Example 2](/Mod1/Images/Week5/ScopeExample2.png)
```

Comment:
This image is one I might change from an accessibility standpoint.  Orange and red are very close to each other on the color wheel and might be more difficult to differentiate than something like orange and green.